portal-api/src/v1/gef/controllers/validation/application.js

```diff
@@ -58,6 +58,9 @@ const validatorStatusCheckEnums = (doc) => {
     case CONSTANTS.DEAL.GEF_STATUS.BANK_CHECK:
     case CONSTANTS.DEAL.GEF_STATUS.SUBMITTED_TO_UKEF:
     case CONSTANTS.DEAL.GEF_STATUS.ABANDONED:
+    // statuses received from TFM - shouldnt be updating to UKEF_ACKNOWLEDGED on its own
+    // eslint-disable-next-line no-fallthrough
+    case CONSTANTS.DEAL.GEF_STATUS.UKEF_ACKNOWLEDGED:
```

Comment:
Sorry @Zainzzkk to avoid having to add the eslint ignore (Didn't realise this would be an issue, can we add a comment at the top instead

```js
// NOTE: UKEF_ACKNOWLEDGED status should only be received by TFM.
switch (doc.status) {
  // ...
}
```

Reply:
done